contrib/langchain4j/src/test/java/com/google/adk/models/langchain4j/LangChain4jTest.java

```diff
@@ -0,0 +1,217 @@
+package com.google.adk.models.langchain4j;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.google.adk.agents.BaseAgent;
+import com.google.adk.agents.LlmAgent;
+import com.google.adk.agents.RunConfig;
+import com.google.adk.events.Event;
+import com.google.adk.runner.InMemoryRunner;
+import com.google.adk.runner.Runner;
+import com.google.adk.sessions.Session;
+import com.google.adk.tools.AgentTool;
+import com.google.adk.tools.Annotations.Schema;
+import com.google.adk.tools.FunctionTool;
+import com.google.genai.types.Content;
+import com.google.genai.types.FunctionCall;
+import com.google.genai.types.FunctionResponse;
+import com.google.genai.types.Part;
+import dev.langchain4j.model.anthropic.AnthropicChatModel;
+import dev.langchain4j.model.openai.OpenAiChatModel;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+
+public class LangChain4jTest {
+
+    public static final String CLAUDE_3_7_SONNET_20250219 = "claude-3-7-sonnet-20250219";
+    public static final String GEMINI_2_0_FLASH = "gemini-2.0-flash";
+    public static final String GPT_4_O_MINI = "gpt-4o-mini";
+
+    @BeforeAll
+    public static void setUp() {
```

Comment:
For this to pass on CI, we'll need to put someone's x3 API keys as secrets into the configuration of this repo.

That's going to be fun to sort out... 🤣